recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,260 @@
+            "mq",
+            "mturk-requester",
+            "neptune",
+            "opsworks",
+            "opsworkscm",
+            "organizations",
+            "pinpoint",
+            "polly",
+            "pricing",
+            "queues",
+            "quicksight",
+            "ram",
+            "rds",
+            "redshift",
+            "recognition",
+            "resource-groups",
+            "robomaker"
+            "route53",
+            "route53domains",
+            "s3",
+            "sagemaker",
+            "sdb",
+            "serverlessrepo"
+            "servicecatalog",
+            "servicediscovery",
+            "shield",
+            "signer",
+            "sms",
+            "snowball",
+            "sns",
+            "sqs",
+            "ssm",
+            "states",
+            "storagegateway",
+            "sts",
+            "support",
+            "swf",
+            "text-to-speech",
+            "texttract",
+            "transcribe",
+            "transfer",
+            "translate",
+            "waf",
+            "workdocs",
+            "worklink",
+            "workmail",
+            "workspaces",
+            "xray"
+           )
+    options = {
+            **{ x: [True, False] for x in sdks},
+            **{
+                "shared": [True, False],
+                "fPIC": [True, False],
+                "min_size": [True, False],
+                }
+            }
+    default_options = {key: False for key in options.keys()}
+    default_options["fPIC"] = True
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+            if (self.settings.compiler == "gcc"
+                    and tools.Version(self.settings.compiler.version) < "6.0"):
+                raise ConanInvalidConfiguration("""Doesn't support gcc5 / shared.
+                See https://github.com/conan-io/conan-center-index/pull/4401#issuecomment-802631744""")
+
+    def requirements(self):
+        self.requires("aws-c-common/0.4.25")
+        self.requires("aws-c-event-stream/0.1.5")
+        self.requires("zlib/1.2.11")
+        if self.settings.os != "Windows":
+            self.requires("libcurl/7.74.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+        # Keeping these unused folders make the source copy fail on Windows
+        shutil.rmtree(self._source_subfolder + "/code-generation/generator")
+        shutil.rmtree(self._source_subfolder + "/aws-cpp-sdk-core-tests")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+
+        build_only = ['core']
```

Comment:
Cosmetic remark: please be consistent in use of quotes and use double quotes everywhere.

Reply:
black format anyone?

Reply:
I've no experience with it, but maybe a github hook doing some checks gets accepted.

Reply:
Replace by double quote